### ISSUE TITLE
yolov3_tiny模型测试与darknet结果不一致

### ISSUE THREAD

**Issue by xiaozhuka** [ChenYingpeng/caffe-yolov3#3]
您好，请问您测试过yolov3_tiny模型吗？我在Windows下测试与darknet结果不一致
已经将分享的代码make_yolo_layer部分针对yolov3_tiny进行了修改，得到的预测结果差异较大
反推回到两个输出层向量有差异

**Comment by claudehang** [comment]
Should I modify any part of the code? I simply just cannot run through but get error "an illegal memory access was encountered". Any advice? Many Thanks!

**Comment by Francis-Xia** [comment]
> 您好，请问您测试过yolov3_tiny模型吗？我在Windows下测试与darknet结果不一致
> 已经将分享的代码make_yolo_layer部分针对yolov3_tiny进行了修改，得到的预测结果差异较大
> 反推回到两个输出层向量有差异

Yes, I also found results are different. Did you solve the problem?
Thanks in advance.

**Comment by MichaelSunEngineer** [comment]
@Francis-Xia 您好，我也想将yolov3-tiny转为caffemodel，但是转时候失败了，请问你是怎么做的，可不可以分享一下？？？不胜感激
